editor/script_editor_debugger.cpp

```diff
@@ -493,23 +493,31 @@ void ScriptEditorDebugger::_parse_message(const String &p_msg, const Array &p_da
 			pinfo.usage = PropertyUsageFlags(int(prop[4]));
 			Variant var = prop[5];
 
+			Variant::Type TYPE = var.get_type();
+
+			if (pinfo.type == Variant::OBJECT) {
+				if (var.is_zero()) {
+					var = RES();
+				} else if (var.get_type() == Variant::STRING) {
+					var = ResourceLoader::load(var);
+
+					if (pinfo.hint_string == "Script")
+						debugObj->set_script(var);
```

Comment:
This is the only line I'm really unsure about, this was the only way I could make scripts show up in the in the inspector but it doesn't look quite right. What do you guys think?